code/__HELPERS/unsorted.dm

```diff
@@ -1210,6 +1210,9 @@ proc/pick_closest_path(value, list/matches = get_fancy_list_of_atom_types())
 #define DELTA_CALC max(((max(world.tick_usage, world.cpu) / 100) * max(Master.sleep_delta,1)), 1)
 
 /proc/stoplag()
+	if (!Master || !(Master.current_runlevel & RUNLEVELS_DEFAULT))
```

Comment:
RUNLEVELS_DEFAULT is setup, ingame, and post game. Doesn't count lobby. Maybe you want just !Master.current_runlevel? because if initializations are done it will always be non-zero